packages/x-components/tests/e2e/cucumber/common-steps.spec.ts

```diff
@@ -173,9 +173,16 @@ Given('an ID results API', () => {
 
 Given('a results API with a known response', () => {
   cy.intercept('https://api.empathy.co/search', req => {
-    req.reply({
+    req.reply(<SearchResponse>{
```

Comment:
I read about this one this weekend instead of using `as SearchResponse`, I like it.